src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -168,6 +170,22 @@ private boolean isNotificationEventValidToSendToAppellant() {
                 || !notificationsMaybeNotForAppellant.contains(getNotificationType());
     }
 
+    private boolean isProcessHearingRequestValidToSend(PartyItemList partyItemList) {
+        return PROCESS_HEARING_RECORDING_REQUEST.equals(getNotificationType()) && isValidHearingRecordingRequestsForCitizen(partyItemList);
+    }
+
+    private boolean isValidHearingRecordingRequestsForCitizen(PartyItemList partyItemList) {
+        final List<HearingRecordingRequest> oldReleasedRecordings;
+        if (responseWrapper.getOldSscsCaseData() != null && responseWrapper.getOldSscsCaseData().getSscsHearingRecordingCaseData().getCitizenReleasedHearings() != null) {
+            oldReleasedRecordings = responseWrapper.getOldSscsCaseData().getSscsHearingRecordingCaseData().getCitizenReleasedHearings();
+        } else {
+            oldReleasedRecordings = new ArrayList<>();
+        }
+        final List<HearingRecordingRequest> newReleasedRecordings = responseWrapper.getNewSscsCaseData().getSscsHearingRecordingCaseData().getCitizenReleasedHearings();
+        final List<HearingRecordingRequest> filteredReleasedRecordings = newReleasedRecordings.stream().filter(e -> !oldReleasedRecordings.contains(e)).collect(Collectors.toList());
```

Comment:
combine this and the next one to one line. 
Optional: Move to a new method to describe what it does, i.e. has new released recordings for party
Mandatory: After stream() move any other statements in its own line so that it's readable.